azureenergylabelerlib/entities.py

```diff
@@ -368,15 +365,8 @@ def exempted_policies(self):
 
     def get_open_findings(self, findings):
         """Findings for the subscription."""
-        if not issubclass(DataFrame, type(findings)):
-            findings = pd.DataFrame([finding.measurement_data for finding in findings])
-        df = findings  # pylint: disable=invalid-name
-        try:
-            open_findings = df[(df['Subscription ID'] == self.subscription_id)]
-        except KeyError:
-            self._logger.info(f'No findings for subscription {self.subscription_id}')
-            open_findings = pd.DataFrame([])
-        return open_findings
+        return [finding.measurement_data for finding in findings
```

Comment:
The `measurement_data` is an attribute we created so the dataframe has the appropriate values to work on. Since pandas is out of the picture why do we still use the `measurement_data` and not the full finding?